app/views/layouts/application.html.erb

```diff
@@ -9,7 +9,30 @@
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
-  <body>
+<body>
+  <div id="flex-content-container">
+    <nav class="navbar navbar-expand-sm navbar-light bg-faded">
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#nav-content" aria-controls="nav-content"
+        aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <!-- Logo -->
+      <a class="navbar-brand" href="#">Logo</a>
+      <!-- Chess Game Links -->
+      <div class="collapse navbar-collapse justify-content-end" id="nav-content">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <a class="nav-link" href="#">New Game</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Sign Up</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Login</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
     <%= yield %>
-  </body>
+</body>
```

Comment:
I think you made a typo that div<flex-content-container> is not closed.

Reply:
@liangma87 The code highlighted in red is deprecated and not included in the code. I just doubled check the request that line is not there.

Reply:
@Claudwin sorry I thought I have typed it out but apparently I didn't.  I was trying to say <div id="flex-content-container"> is not closed.

Reply:
@liangma87 Thanks submitting a pull request now